packages/release/release.spec

```
@@ -198,6 +206,12 @@ install -p -m 0644 %{S:1104} %{buildroot}%{_cross_unitdir}/service.d/00-aws-conf
 install -d %{buildroot}%{_cross_libdir}/systemd/system.conf.d
 install -p -m 0644 %{S:98} %{buildroot}%{_cross_libdir}/systemd/system.conf.d/80-release.conf
 
+install -d %{buildroot}%{_cross_libdir}/modules-load.d
+install -p -m 0644 %{S:100} %{buildroot}%{_cross_libdir}/modules-load.d/nf_conntrack.conf
+
+install -d %{buildroot}%{_cross_libdir}/systemd/journald.conf.d
+install -p -m 0644 %{S:101} %{buildroot}%{_cross_libdir}/systemd/journald.conf.d/80-journald.conf
```

The previous path is hard-coded in a `bloodhound` check (`JOURNALD_CONF_FILE`) so either that implementation needs to be smarter or we should keep the path the same:
```suggestion
install -p -m 0644 %{S:101} %{buildroot}%{_cross_libdir}/systemd/journald.conf.d/journald.conf
```

We should really be using `systemd-analyze cat-config systemd/journald.conf` there but that's more of an issue for the backlog.

---

Gotcha! I was trying to avoid a situation like we saw recently with the `max-delay` in logind